codec-http/src/main/java/io/netty/handler/codec/http/HttpMethod.java

```diff
@@ -85,8 +85,7 @@
      */
     public static final HttpMethod CONNECT = new HttpMethod("CONNECT");
 
-    private static final Map<String, HttpMethod> methodMap =
-            new HashMap<String, HttpMethod>();
+    private static final Map<String, HttpMethod> methodMap = new TreeMap<String, HttpMethod>();
```

Comment:
Good catch. I was just experimenting.  The lookup in this map was showing up as a hot path, but changing the map didn't really alleviate the pain.